packages/core/crates/core-network/src/ping.rs

```diff
@@ -34,62 +38,60 @@ use utils_misc::time::wasm::current_timestamp;
 
 const PINGS_MAX_PARALLEL: usize = 14;
 
+// TODO: NOTE: UnboundedSender and UnboundedReceiver are bound only by available memory
+// in case of faster input than output the memory might run out
+pub type HeartbeatSendPingTx = futures::channel::mpsc::UnboundedSender<(PeerId, ControlMessage)>;
```

Comment:
Please make these bounded and we can perhaps make these configurable in the future. 
Pings should be discarded until the queues free up.

Reply:
Sure, thay's why I left a TODO, thid can change.

Reply:
This queue can stay bounded, as the queue generating the requests before this is bounded.